pkg/bootstrap/option/convert.go

```diff
@@ -180,6 +181,32 @@ func durationConverter(value *types.Duration) convertFunc {
 	}
 }
 
+// openCensusAgentContextConverter returns a converter that returns the list of
+// distributed trace contexts to propagate with envoy.
+func openCensusAgentContextConverter(contexts []meshAPI.Tracing_OpenCensusAgent_TraceContext) convertFunc {
+	return func(*instance) (interface{}, error) {
+		if len(contexts) == 0 {
+			return convertToJSON([]string{"TRACE_CONTEXT", "GRPC_BIN", "CLOUD_TRACE_CONTEXT", "B3"}), nil
```

Comment:
probably doesn't make a difference, but should we just have this be a preconverted const or var (something like AllContexts) ?

Reply:
SG. I wanted to keep the scope of envoy trace contexts to just this function so it's easy to extract out/modify when the new telemetry api starts to land. Made it into a pre-converted var in the function.